fed/_private/grpc_options.py

```diff
@@ -30,29 +31,43 @@
 _GRPC_MAX_SEND_MESSAGE_LENGTH = _DEFAULT_GRPC_MAX_SEND_MESSAGE_LENGTH
 _GRPC_MAX_RECEIVE_MESSAGE_LENGTH = _DEFAULT_GRPC_MAX_RECEIVE_MESSAGE_LENGTH
 
+logging.info(f"global {_GRPC_MAX_SEND_MESSAGE_LENGTH} {_GRPC_MAX_RECEIVE_MESSAGE_LENGTH}")
```

Comment:
[memo / fixme]
Module reloading is occurred, that means it seems to be what's causing the bug, resetting the global variables.